modules/core/test/v2/unit/wallet.ts

```diff
@@ -590,6 +590,42 @@ describe('V2 Wallet:', function() {
     }));
   });
 
+  describe('maxNumInputsToUse verification', function() {
+    const address = '5b34252f1bf349930e34020a';
+    const maxNumInputsToUse = 2;
+    let basecoin;
+    let wallet;
+
+    before(co(function* () {
+      basecoin = bitgo.coin('tbtc');
+      const walletData = {
+        id: '5b34252f1bf349930e34020a',
+        coin: 'tbtc',
+        keys: [
+          '5b3424f91bf349930e340175'
+        ]
+      };
+      wallet = new Wallet(bitgo, basecoin, walletData);
+    }));
+
+
+    it('should pass maxNumInputsToUse parameter when calling fanout unspents', co(function* () {
```

Comment:
same

Reply:
I've added a second commit that replaces promises in this test with async-await.

Reply:
:+1: 

both the Bluebird coroutine-wrapper and the `async` keyword use Promises, the latter is just nicer